code.gs

```diff
@@ -1659,6 +1659,37 @@ var i18n = {
     'It looks like you are using an outdated version of this script': 'Parece que tens uma versão desatualizada deste script',
     'You can find the latest one here': 'Podes encontrar a última versão aqui',
   },
+  'fa': {
+    'Age': 'سن',
+    'Years': 'سال',
+    'Events': 'رویدادها',
+    'Birthdays today': 'تولدهای امروز',
+    'Birthdays tomorrow': 'تولدهای فردا',
+    'Birthdays in {0} days': 'تولدهای {0} روز آینده',
+    'Anniversaries today': 'سالگردهای امروز',
+    'Anniversaries tomorrow': 'سالگردهای فردا',
+    'Anniversaries in {0} days': 'سالگردهای {0} روز آینده',
+    'Custom events today': 'رویدادهای شخصی امروز',
+    'Custom events tomorrow': 'رویدادهای شخصی فردا',
+    'Custom events in {0} days': 'رویدادهای شخصی {0} روز آینده',
+    'Hey! Don\'t forget these events': 'سلام! این رویدادها را فراموش نکن',
+    'version': 'نسخه',
+    'dd-MM-yyyy': 'dd-MM-yyyy',
+    'Mobile phone': 'شماره موبایل',
+    'Work phone': 'شماره تلفن محل کار',
+    'Home phone': 'شماره تلفن خانه',
+    'Main phone': 'شماره تلفن اصلی',
+    'Other phone': 'شماره تلفن دیگر',
+    'Home fax': 'شماره فاکس خانه',
+    'Work fax': 'شماره فاکس محل کار',
+    'Google voice': 'وویس گوگل',
+    'Pager': 'پیجر',
+    'Home email': 'ایمیل خانه',
+    'Work email': 'ایمیل محل کار',
+    'Other email': 'ایمیل دیگر',
+    'It looks like you are using an outdated version of this script': 'به نظر می رسد شما نسخه قدیمی این اسکریپت را استفاده می کنید',
+    'You can find the latest one here': 'اینجا می توانید نسخه به روز را بیابید',
+},
```

Comment:
Could you please add two spaces before the closed curly brace?

Reply:
Sure!